doc/cookbook/source/references.bib

```diff
@@ -163,3 +163,10 @@ @article{hyvarinen2000independent
   pages={411--430},
   year={2000}
 }
+@inproceedings{basak2008kernel
+	author={J. Basak},
+	booktitle={2008 19th International Conference on Pattern Recognition},
+	title={A least square kernel machine with box constraints},
+	year={2008},
+	pages={1-4},
```

Comment:
pls do proper references. These pages here are definitely not correct

Reply:
ICPR papers are not re-assigned a page number when they are published. This is based on the reference format provided in https://ieeexplore.ieee.org/document/4761717

Reply:
I see, maybe we can remove them then?